src/video/SDL_surface.c

```diff
@@ -199,6 +204,15 @@ SDL_CreateSurfaceFrom(void *pixels,
         return NULL;
     }
 
+    if (SDL_ISPIXELFORMAT_FOURCC(format)) {
+        int expected_pitch = 0;
+        SDL_SW_SizeYUV(format, width, height, &expected_pitch);
+        if (expected_pitch != pitch) {
+            SDL_SetError("Only accept exact pitch for YUV format");
```

Comment:
Why do we need exact pitch? SDL_ConvertPixels() takes the surface pitches into account...?

Reply:
I've removed it